src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/DocumentState.ComputedStateTracker.cs

```diff
@@ -0,0 +1,253 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Razor.Language;
+
+namespace Microsoft.CodeAnalysis.Razor.ProjectSystem;
+
+internal partial class DocumentState
+{
+    // Internal, because we are temporarily sharing code with CohostDocumentSnapshot
+    private class ComputedStateTracker
```

Comment:
Comment seems wrong, as this is private, not internal.

Also, did this class change other than the `forceRuntimeCodeGeneration`? Its just a big block of green on GitHub sadly, despite your best efforts

Reply:
ah yea, I changed it to private and moved the internal statics to `DocumentState`. Otherwise no changes

Reply:
ICYMI, the comment above sats "internal", and can probably just be removed, since clearly making it private didn't break anything.

Reply:
Yeah, that comment should probably just be removed.